libs/pilight/core/json.c

```diff
@@ -1436,3 +1436,26 @@ int json_find_string(JsonNode *object, const char *name, char **out) {
 void json_free(void *a) {
 	free(a);
 }
+
+JsonNode *json_vartype(int vartype) {
+	JsonNode *vartype_array = json_mkarray();
+	if ((vartype & JSON_NULL) != 0) {
```

Comment:
Right after the `if`, there should be no space before the opening
bracket `(`. So change: `if (` to `if(`